src/net.cpp

```diff
@@ -1996,15 +1998,17 @@ void StartNode(void* parg)
     util::ThreadSetInternalName("grc-nodestart");
 
     fShutdown = false;
-    MAX_OUTBOUND_CONNECTIONS = (int)gArgs.GetArg("-maxoutboundconnections", 8);
+    MAX_OUTBOUND_CONNECTIONS = (int) gArgs.GetArg("-maxoutboundconnections", 8);
+    int max_connections = std::min<int>(gArgs.GetArg("-maxconnections", 125), 950);
     int nMaxOutbound = 0;
     if (semOutbound == nullptr) {
         // initialize semaphore
-        nMaxOutbound = min(MAX_OUTBOUND_CONNECTIONS, (int)gArgs.GetArg("-maxconnections", 125));
+        nMaxOutbound = std::min<int>(MAX_OUTBOUND_CONNECTIONS, max_connections);
         semOutbound = new CSemaphore(nMaxOutbound);
     }
 
-    LogPrintf("Using %i OutboundConnections with a MaxConnections of %" PRId64, MAX_OUTBOUND_CONNECTIONS, gArgs.GetArg("-maxconnections", 125));
+    LogPrintf("Using %i OutboundConnections with a MaxConnections of %" PRId64,
+              MAX_OUTBOUND_CONNECTIONS, max_connections);
```

Comment:
Same as mentioned above
"Using 8 OutboundConnections with a MaxConnections of 5"

Reply:
See above.

Reply:
I think we can solve this by using nMaxOutbound instead of MAX_OUTBOUND_CONNECTIONS in the logging statement.